app/models/resource_action.rb

```diff
@@ -5,6 +5,10 @@ class ResourceAction < ApplicationRecord
 
   serialize  :ae_attributes, Hash
 
+  PROVISION   = 'Provision'.freeze
+  RETIREMENT  = 'Retirement'.freeze
+  RECONFIGURE = 'Reconfigure'.freeze
```

Comment:
Yes there are. I think it better to have another PR to find/replace all the uses of the keywords so that those changes do not mix with the files for Ansible Playbook only.